ocaml/xapi/test_vdi_cbt.ml

```diff
@@ -524,25 +524,20 @@ let test_vdi_list_changed_blocks () =
   let vdi_to = Test_common.make_vdi ~__context~location:vdi_to_location  ~sR ~cbt_enabled:true ~managed:true () in
 
   register_smapiv2_server
-    ~vdi_list_changed_blocks:(fun _ ~dbg ~sr ~vdi_from ~vdi_to -> list_changed_blocks_params := Some (sr,vdi_from,vdi_to); list_changed_blocks_string)
+    ~vdi_list_changed_blocks:(fun _ ~dbg ~sr ~vdi_from ~vdi_to -> list_changed_blocks_params := Some (sr,(vdi_from,vdi_to)); list_changed_blocks_string)
     (Db.SR.get_uuid ~__context ~self:sR);
 
-  OUnit.assert_equal
-    ~msg:(Printf.sprintf "VDI.list_changed_blocks should return %s" list_changed_blocks_string)
+  Alcotest.(check string) "VDI.list_changed_blocks"
     (Xapi_vdi.list_changed_blocks ~__context ~vdi_from ~vdi_to)
     list_changed_blocks_string;
-  OUnit.assert_equal
-    ~msg:("Incorrect parameters passed")
-    (Some (sr_uuid, vdi_from_location, vdi_to_location))
+  Alcotest.(check (option (pair string (pair string string)))) "VDI.list_changed_blocks parameters"
+    (Some (sr_uuid, (vdi_from_location, vdi_to_location)))
     !list_changed_blocks_params
 
 let test =
-  let open OUnit in
-  "test_vdi_cbt" >:::
-  [ "test_cbt_enable_disable" >:: test_cbt_enable_disable
-  ; "test_set_metadata_of_pool_doesnt_allow_cbt_metadata_vdi" >:: test_set_metadata_of_pool_doesnt_allow_cbt_metadata_vdi
-  ; test_get_nbd_info
-  ; "test_allowed_operations_updated_when_necessary" >:: test_allowed_operations_updated_when_necessary
-  ; test_data_destroy
-  ; "test_vdi_list_changed_blocks" >:: test_vdi_list_changed_blocks
-  ]
+  [ "test_cbt_enable_disable", `Quick, test_cbt_enable_disable
+  ; "test_set_metadata_of_pool_doesnt_allow_cbt_metadata_vdi", `Quick, test_set_metadata_of_pool_doesnt_allow_cbt_metadata_vdi
+  ; "test_allowed_operations_updated_when_necessary", `Quick, test_allowed_operations_updated_when_necessary
+  ; "test_vdi_list_changed_blocks", `Quick, test_vdi_list_changed_blocks ]
+  @ test_get_nbd_info
```

Comment:
I agree that long lines aren't the best here, maybe split them like so:
```
"test_cbt_enable_disable",
  `Quick, test_cbt_enable_disable
```
or
```
"test_cbt_enable_disable",
  `Quick,
  test_cbt_enable_disable
```
whichever you think is best?

Reply:
In general I would agree with your comment but I think that for the tests lists it is actully more readable to keep them long and self-contained. It's not that we are going to touch the tests so much after all

Reply:
True - so I guess I can merge now this as it is.